client/state/sites/selectors/get-seo-title-formats-for-site.js

```diff
@@ -17,7 +12,4 @@ import { fromApi as seoTitleFromApi } from 'calypso/components/seo/meta-title-ed
  * @param  {object} site Selected site
  * @returns {object} Formats by type e.g. { frontPage: { type: 'siteName' } }
  */
-export default compose(
-	seoTitleFromApi,
-	partialRight( get, 'options.advanced_seo_title_formats', {} )
-);
+export default ( site ) => seoTitleFromApi( site?.options?.advanced_seo_title_formats ?? {} );
```

Comment:
😞  This isn't really a selector. Seems like it would be a good idea to move it out of this directory.

Also, it feels off to have to pass the entire site object when we can just pass the ID and get the rest out of the state. Maybe this is worth refactoring to an actual selector?

Reply:
That's true, and `buildSeoTitle` is also not a selector. I'd rather keep this SEO code as is and maybe refactor it on some other occasion.

Reply:
This PR is pretty large already, so sounds fine by me.